artifacts/mirroring-staging/filestores/k8s-staging-cri-tools/filepromoter-manifest.yaml

```diff
@@ -0,0 +1,7 @@
+filestores:
+- base: gs://k8s-artifacts-prod/binaries/cri-tools/
+  src: true
+- base: s3://test-artifacts-k8s-io-us-east-2/binaries/cri-tools/
```

Comment:
do we have auth setup such that this will work?

Reply:
So this can work without auth, it's all publicly accessible (currently).  I am not sure whether it will try to get an auth token and fail or whether it will simply not try to get one (and then hopefully succeed).

I'm also hoping we'll find out now that you're merged the test-infra job - thank you!